src/System.Text.JsonLab/System/Text/Json/JsonWriterUtf8.cs

```diff
@@ -565,12 +565,9 @@ public void WriteValue(Guid value)
         /// </summary>
         public void WriteNull()
         {
-            //TODO: Optimize, just like WriteValue(long value)
-            WriteItemSeperatorUtf8();
-            _firstItem = false;
-            WriteSpacingUtf8();
-
-            WriteJsonValue(JsonConstants.NullValue);
+            ReadOnlySpan<byte> valueSpan = JsonConstants.NullValue;
+            int bytesNeeded = CalculateValueBytesNeeded(valueSpan, sizeof(byte), 2);
+            WriteValueUtf8(valueSpan, bytesNeeded);
```

Comment:
I don't think calling the existing WriteValueUtf8 is correct here. This method writes the value surrounded by quotations. We only want the characters `n u l l` to be written. You should add your own method, like WriteNullUtf8 for now (or re-write WriteJsonValue).